infrastructure/lambda/api.yaml

```diff
@@ -436,4 +462,6 @@ x-amazon-apigateway-request-validators:
   Validate both:
     validateRequestBody: true
     validateRequestParameters: true
+  Validate header:
+    validateRequestParameters: true
```

Comment:
the name of this validator feels like it doesn't match up to validating request params